source/guides/guides/cross-browser-testing.md

```diff
@@ -2,7 +2,7 @@
 title: Cross Browser Testing
 ---
 
-Cypress has the capability to run tests across multiple browsers. Currently, Cypress has support for {% url "Chrome-family browsers" launching-browsers#Chrome-Browsers %} (including Electron) and beta support for Firefox browsers.
+Cypress has the capability to run tests across multiple browsers. Currently, Cypress has support for {% url "Chrome-family browsers" launching-browsers#Chrome-Browsers %} (including Electron) and {% url "beta support for Firefox browsers and Microsoft Edge" https://circleci.com/orbs/registry/orb/cypress-io/cypress %}.
```

Comment:
Curious, why link Cypress orb for mentioning FF and Edge?

Reply:
That's my fault. ;)  I meant to link https://www.cypress.io/blog/2020/02/06/introducing-firefox-and-edge-support-in-cypress-4-0/, but I must have copy/pasted the wrong them.  New commit on its way...